app/services/ui_notifications/rss_notifications_checker.rb

```diff
@@ -70,5 +72,18 @@ def rss_notification_blueprint
     def notification_already_exists?(item)
       !!Notification.unscoped.find_by_message(item.title)
     end
+
+    def load_rss_feed
+      uri = URI.parse(@url)
+      Net::HTTP.start(uri.host, uri.port, :use_ssl => uri.scheme == 'https') do |http|
+        request = Net::HTTP::Get.new(uri.request_uri)
+        request.initialize_http_header({"User-Agent" => rss_user_agent})
+        http.request(request).body
```

Comment:
Should there be any error checking for incorrect HTTP status codes? It's possible the host is unreachable, misconfigured or a proxy is refusing it.